nasa/cumulus
# CUMULUS-3876: Fix S3 Replicator cross region bucket writes

**Summary:** Summary of changes

Addresses [CUMULUS-3876: S3 Replicator cross region bucket writes failing](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3876) 

## Changes

* Fixed `s3-replicator` lambda cross region write failure
* Added `target_region` variable to `tf-modules/s3-replicator` module

## Test
* rename s3-replicator.tf.example to s3-replicator.tf, deply example/cumulus-tf
* add the variable to terraform.tfvars
s3_replicator_config = {
  source_bucket = "cumulus-test-sandbox-access-logs"
  source_prefix = "s3_access_logs"
  target_bucket = "cumulus-test-sandbox-access-logs-destination"
  target_prefix = "input/s3_access/sandbox"
  target_region = "us-west-2"
}
* add a file to the source folder, the file should be replicated to destination bucket folder

## PR Checklist
